assignments/objects.js

```diff
@@ -14,31 +14,73 @@ let example = {
   "id": 0,
   "name": "Example",
   "email": "examples@you.edu",
-  "gender": "F"
+  "gender": "F" 
 }
 
 // Write your intern objects here:
-
+const internInfo1 = 
+    {
+      "id":1,
+      "name": "Mitzi,F",
+      "email": "mmelloy0@psu.edu",
+      "gender": "F",
+    };
+const internInfo2 =
+    {
+      "id": 2,
+      "name": "Kennan,M",
+      "email": "kdiben1@tinypic.com",
+      "gender": "M",
+      "speak": function(){
+          return `Hello, my name is ${internInfo2.name} `
+      }
+    };
+const internInfo3 =
+    {
+      "id":3,
+      "name": "Keven,M",
+      "email": "kmummery2@wikimedia.org",
+      "gender": "M",
+    };
+const internInfo4 =
+    {
+      "id":4,
+      "name": "Gannie,M",
+      "email": "gmartinson3@illinois",
+      "gender": "M",
+    };
+const internInfo5 =
+    {
+      "id":5,
+      "name": "Antonietta,F",
+      "email": "adaine5@samsung.com",
+      "gender": "F",
+      "multiplyNums": function(a,b){
+        return a * b;
+      }
+    };
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(internInfo1.name);
 // Kennan's ID
-
+console.log(internInfo2.id);
 // Keven's email
-
+console.log(internInfo3.email);
 // Gannie's name
-
+console.log(internInfo4.name);
 // Antonietta's Gender
+console.log(internInfo5.gender);
+
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(internInfo2.speak());
```

Comment:
Try assignment, I like how you figured out how to nest this but another way is to... `internInfo2.speak = function...`